app/Resources/views/base.html.twig

```diff
@@ -49,7 +49,23 @@
                         </div>
                         <div class="navbar-collapse collapse">
                             <ul class="nav navbar-nav navbar-right">
+                                {# Generate a dropdown with all our locales #}
+                                <li class="dropdown">
+                                    <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false"><i class="fa fa-flag"></i> <span class="caret"></span></a>
+                                    <ul class="dropdown-menu locales" role="menu">
+                                        <li><a href="{{ path('blog_index', {_locale:'de'}) }}">Deutsch</a></li>
+                                        <li><a href="{{ path('blog_index', {_locale:'en'}) }}">English</a></li>
+                                        <li><a href="{{ path('blog_index', {_locale:'es'}) }}">Español</a></li>
+                                        <li><a href="{{ path('blog_index', {_locale:'fr'}) }}">Français</a></li>
+                                        <li><a href="{{ path('blog_index', {_locale:'ro'}) }}">Român</a></li>
+                                        <li><a href="{{ path('blog_index', {_locale:'ru'}) }}">Pусский</a></li>
+                                        <li><a href="{{ path('blog_index', {_locale:'uk'}) }}">Yкраїнська</a></li>
+                                        <li><a href="{{ path('blog_index', {_locale:'cs'}) }}">česká republika</a></li>
```

Comment:
I guess `česká republika` means `Czech Republic` (the country) instead of `Czech` (the language). Is it `Česko` more appropriate?